InternetHospital.WebApi/Controllers/AppointmentsController.cs

```diff
@@ -44,92 +42,56 @@ public IActionResult GetAvailableAppointments([FromQuery] int doctorId)
         [HttpPost("create")]
         public IActionResult AddAppointment([FromBody] AppointmentCreationModel creationModel)
         {
-            if (int.TryParse(User.Identity.Name, out var doctorId))
-            {
-                var (status, message) = _appointmentService.AddAppointment(creationModel,doctorId);
-
-                if (status)
-                {
-                    return Ok(new { message });
-                }
-                else
-                {
-                    return BadRequest(new { message });
-                }
-            }
-            else
+            if (!int.TryParse(User.Identity.Name, out var doctorId))
             {
                 return BadRequest(new { message = "Wrong claims" });
             }
+
+            var (status, message) = _appointmentService.AddAppointment(creationModel, doctorId);
+
+            return status ? (IActionResult)Ok(new { message }) : BadRequest(new { message });
```

Comment:
Why you have used explicit conversion to IActionResult and only for Ok()?

Reply:
There is no implicit conversion between OkObjectResult and BadRequestObjectResult